cpp/locality/BondHistogramCompute.h

```diff
@@ -122,7 +122,7 @@ class BondHistogramCompute
         m_local_histograms; //!< Thread local bin counts for TBB parallelism
 
     using BondHistogram = util::Histogram<unsigned int>;
-    using BHAxes = typename BondHistogram::Axes;
+    using BHAxes = typename util::Axes;
```

Comment:
I don't think the `typename` is needed anymore, but please check to make sure this compiles.
```suggestion
    using BHAxes = util::Axes;
```

A slightly more useful refactoring could be to find-replace `BHAxes` with `util::Axes` and remove this definition here. The only reason we needed the type `BHAxes` was that this was previously a dependent type of `util::Histogram<unsigned int>`, and it is no longer a dependent type with the changes in this PR.